$OpenBSD: patch-tests_issue1875-honor-no-set-default_sh,v 1.1 2010/11/02 20:22:35 kili Exp $

Our find(1) doesn't exit(1) if a path operand couldn't be traversed,

--- tests/issue1875-honor-no-set-default.sh.orig	Mon Aug 23 20:35:23 2010
+++ tests/issue1875-honor-no-set-default.sh	Sat Aug 28 20:27:21 2010
@@ -28,7 +28,7 @@
 rm -rf R S                      # Another script may have left a mess.
 darcs init      --repo R        # Create our test repos.
 darcs get R S  --no-set-default
-not find S/_darcs/prefs/defaultrepo
+test ! -e S/_darcs/prefs/defaultrepo
 rm -rf S
 
 darcs init --repo S
